test/unit/routers/alpha-router/functions/get-candidate-pools.test.ts

```diff
@@ -170,6 +170,34 @@ describe('get candidate pools', () => {
     ).toBeTruthy();
   });
 
+  test('succeeds to get top pools while excluding WETH', async () => {
```

Comment:
Should we also add a test for the scenario where WETH would not be excluded (asking route as Acc_inv meaning no tokens need to be excluded)

Reply:
The other tests in the file do exactly that :D